pulsar-package-management/bookkeeper-storage/src/test/java/org/apache/pulsar/packages/management/storage/bookkeeper/bookkeeper/test/BookKeeperClusterTestCase.java

```diff
@@ -191,7 +191,7 @@ protected String getMetadataServiceUri(String ledgersRootPath) {
         return zkUtil.getMetadataServiceUri(ledgersRootPath);
     }
 
-    @AfterMethod
+    @AfterMethod(alwaysRun = true)
```

Comment:
Should we need do same for `managed-ledger/src/test/java/org/apache/bookkeeper/test/BookKeeperClusterTestCase.java`

 relate issue: 
- https://github.com/apache/pulsar/issues/16863